Packs/CommonPlaybooks/Playbooks/playbook-Search_For_Hash_In_Sandbox_-_Generic.yml

```diff
@@ -663,11 +676,506 @@ tasks:
             value:
               simple: pdf
           ignorecase: true
+    view: |-
+      {
+        "position": {
+          "x": 1830,
+          "y": 695
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "26":
+    id: "26"
+    taskid: 157ba2a0-d8e3-4d38-8360-d1dac15d5ec6
+    type: condition
+    task:
+      id: 157ba2a0-d8e3-4d38-8360-d1dac15d5ec6
+      version: -1
+      name: Is Joe Sandbox enabled?
+      description: Returns 'yes' if the CrowdStrike Falcon X integration is available.
```

Comment:
Is this description correct?  Should it be Joe Sandbox.  And is it Joe Sandbox or JoeSecurity Sandbox?

Reply:
It should be JoeSecurity Sandbox (or just Joe Security). Should I change it or will you take it?

Reply:
NVM, changed that already.